lib/Resource.js

```diff
@@ -28,16 +28,45 @@ var Resource = function(options) {
   this.app = options.app;
   this.sequelize = options.sequelize;
   this.model = options.model;
-  this.include = options.include.map(function(include) {
-    return include instanceof options.sequelize.Model ? { model: include } : include;
+  this.include = options.include.map(function (include) {
```

Comment:
please try to avoid whitespace commits like these, I think they just clutter the PR when we should be focusing on the real changes below.